backend/workflow_manager/endpoint/destination.py

```diff
@@ -155,20 +155,19 @@ def handle_output(
         file_name: str,
         file_hash: FileHash,
         workflow: Workflow,
+        input_file_path: str,
         error: Optional[str] = None,
-        input_file_path: Optional[str] = None,
+        use_file_history: bool = True,
```

Comment:
@chandrasekharan-zipstack wouldn't it be better to keep this `False`? We will basically need this only for the API calls during demo right?

Or is this method applicable for both ETL and API?